cyences_app_for_splunk/default/data/ui/views/cs_asset_intelligence.xml

```diff
@@ -429,10 +430,11 @@
       <table>
         <title>Endpoints Suspended from Central Management</title>
         <search>
-          <query>`cs_sophos` type="Event::Endpoint::Management::*" | rename dhost as host | eval ip=src_ip | search $tkn_filter_main$ | stats count, latest(_time) as _time, values(src_ip) as src_ip, values(suser) as user, latest(type) as type by host
-| where type=="Event::Endpoint::Management::Suspended"
-| `cs_human_readable_time_format(_time, "Suspended from Management Since")`
-| table host, src_ip, user, "Suspended from Management Since"</query>
+          <query>index=main sourcetype="sophos*" type IN ("Event::Endpoint::Management::*") host="*" location IN ("*") source IN ("*") |stats count, latest(_time) as _time, values(source_info.ip) as source_info.ip, values(source) as source by host, location, type
```

Comment:
Use the macro for the index and try to find the exact source type.
index=main sourcetype="sophos*" => cs_sophos sourcetype="sophos_events"

Do not required this filters => host="" location IN ("") source IN ("*")
instead use this token for the filter => | search $tkn_filter_main$
 (verify the field names in token value)
 
why we have added 2 additional fields (location, type) in by clause?